providers/aws/provider/provider.go

```diff
@@ -123,20 +142,24 @@ func (s *Service) connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 	var err error
 
 	switch conf.Type {
-	case SshConnectionType:
```

Comment:
I think `SshConnectionType` and `RegistryImageConnectionType` can be completely removed here, as I did here:
https://github.com/mondoohq/cnquery/pull/1773

cnquery detects, that the os provider should handle these connection types, and automatically uses it. So, no need to add it here.